File: src/main/java/no/nav/pensjon/regler/domain/kode/SatsTypeCti.java
package no.nav.pensjon.regler.domain.kode;

import java.io.Serializable;

public class SatsTypeCti extends TypeCti implements Serializable {

    private static final long serialVersionUID = 6005472676083475303L;

    public SatsTypeCti(TypeCti typeCti) {
        super(typeCti);
    }

    public SatsTypeCti() {
    }

    public SatsTypeCti(String kode) {
        super(kode);
    }

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/kode/SatsTypeCti.kt
package no.nav.pensjon.regler.domain.kode

import java.io.Serializable

class SatsTypeCti : TypeCti, Serializable {
    constructor(typeCti: TypeCti) : super(typeCti)
    constructor()
    constructor(kode: String?) : super(kode)
}